syz-manager/manager.go

```diff
@@ -886,6 +887,19 @@ func (mgr *Manager) runInstanceInner(index int, instanceName string, injectLog <
 	return rep, vmInfo, nil
 }
 
+func prependExecuting(rep *report.Report, lastExec []ExecRecord) {
+	buf := new(bytes.Buffer)
+	fmt.Fprintf(buf, "last executing test programs:\n\n")
```

Comment:
Now that the `executing program X` output is suddenly no longer interleaved with the console output, I think we should be more explicit about that.

Maybe if we 
1) Put a more detailed text describing that these are just the last executed programs before the crash.
2) Put it all after the console output

it will be somewhat more clear?